docs/conf.py

```diff
@@ -78,22 +82,25 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 # html_static_path = ['_static']
 
+html_logo = str(ROOT / 'images/logo.svg')
+html_favicon = str(ROOT / 'images/favicon.svg')
```

Comment:
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_favicon says it should be relative the configuration file and/or stored in _static?

Reply:
Agree. However, `_static` is not used in this repo. Shall I move `docs/images` to `docs/_static/images`?

See https://github.com/hdl/containers/tree/main/doc/_static.

Reply:
Changed in https://github.com/SymbiFlow/symbiflow-examples/pull/253/commits/801bfe59fe5b6ba2ab666733c281452d140ab87a.